src/js/stores/search.js

```diff
@@ -227,7 +229,7 @@ define(function (require, exports, module) {
                     shortenedPaths = pathUtil.getShortestUniquePaths(ancestors).toJS();
 
                 var itemMap = items.map(function (item, index) {
-                    var newPathInfo = shortenedPaths[index] || item.pathInfo;
+                    var newPathInfo = searchTypeInfo.shortenPaths ? shortenedPaths[index] : item.pathInfo;
```

Comment:
So I see a logic change here, and I have to ask. You were checking to see if shortenedPath existed, and then used `item.pathInfo`. What if you set shortenPaths to true, but shortenedPath does not exist for that item? Can you guarantee that won't happen?